models/fivetran_log__mar_table_history.sql

```diff
@@ -27,50 +27,37 @@ ordered_mar as (
         schema_name,
         table_name,
         destination_id,
-        measured_date,
         measured_month,
-        incremental_rows,
-        coalesce(case when lower(free_type) = 'paid'
+        max(measured_date) as last_measured_at,
+        sum(incremental_rows) as incremental_rows,
+        sum(coalesce(case when lower(free_type) = 'paid'
             then incremental_rows
-        end, 0) as paid_monthly_active_rows,
-        coalesce(case when lower(free_type) != 'paid'
+        end, 0)) as paid_monthly_active_rows,
+        sum(coalesce(case when lower(free_type) != 'paid'
             then incremental_rows
-        end, 0) as free_monthly_active_rows
+        end, 0)) as free_monthly_active_rows,
+        (free_monthly_active_rows + paid_monthly_active_rows) as total_monthly_active_rows
 
     from incremental_mar
-
-),
-
-latest_mar as (
-    select 
-        connector_name,
-        schema_name,
-        table_name,
-        destination_id,
-        measured_month,
-        date(measured_date) as last_measured_at,
-        free_monthly_active_rows,
-        paid_monthly_active_rows,
-        (free_monthly_active_rows + paid_monthly_active_rows) as total_monthly_active_rows
-    
-    from ordered_mar
+    {{dbt_utils.group_by(5)}}
 
 ),
 
 mar_join as (
 
-    select 
-        latest_mar.*,
+    select
+        ordered_mar.*,
```

Comment:
Since we will need to keep the latest_mar cte.
```suggestion
        latest_mar.*,
```